.github/workflows/unittests.yml

```diff
@@ -78,6 +78,11 @@ jobs:
             python: "3.10"
             mode: root
             flags: " -K scanner"
+          # Linux with vcan root tests
+          - os: ubuntu-18.04
+            python: "3.10"
+            mode: root
+            flags: " -K scanner"
```

Comment:
You got an idea of why this vcan isn't on recent ubuntus?

I'm not sure we need to keep the 3.10-root test on ubuntu-22.04 if we're adding this one.

Reply:
Unfortunately not. I also didn't found a way to get vcan installed on the latest ubuntu. Therefore, it's necessary to enable an older version of ubuntu

Reply:
Would it be possible to only run vcan tests on this one then?

Reply:
yes, that would be possible